src/NodeApi/Runtime/NativeLibrary.cs

```diff
@@ -1,13 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-#if !NET7_0_OR_GREATER
+#if !NETCOREAPP3_0_OR_GREATER
 
 using System;
+using System.ComponentModel;
 using System.Runtime.InteropServices;
-#if !(NETFRAMEWORK || NETSTANDARD)
-using SysNativeLibrary = System.Runtime.InteropServices.NativeLibrary;
-#endif
+using System.Runtime.InteropServices.ComTypes;
```

Comment:
nit: is it being used? ` System.Runtime.InteropServices.ComTypes`

Reply:
No. VS keeps adding it when I copy and paste... something. And I forgot this one time to remove it. =/

Reply:
Ouch... :)